lib/runner.js

```diff
@@ -63,12 +63,11 @@ function setupRemoteSessionAndRunTests(config, options) {
             browserName: options.browser,
             platform: "ANY"
         };
-    browserOptions['browserstack.local'] = options.browserstackLocal;
     let bsOptions = {
```

Comment:
Could this not be done with something along the lines of an `Object.assign(standardOptions, config)`?